frontend/projects/ui/src/app/services/patch-db/data-model.ts

```diff
@@ -25,12 +25,11 @@ export interface UIData {
 }
 
 export interface UIMarketplaceData {
-  'selected-id': string | null
+  'selected-id': string
```

Comment:
```suggestion
  'selected-url': string
```
this change will require a BE migration